assets/src/dashboard/utils/getRandomNumber.js

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
Let's get rid of this, no need for it since the `label` is surrounding the `input`

Reply:
sure thing